scipy/special/tests/test_basic.py

```diff
@@ -3943,6 +3943,7 @@ def test_some_values(self):
         assert_allclose(special.struve(-8.01, 21), 0.0398716951023, rtol=1e-8)
         assert_allclose(special.struve(-3.0, 200), 0.0142134427432, rtol=1e-12)
         assert_allclose(special.struve(-8.0, -41), 0.0192469727846, rtol=1e-11)
+        assert_allclose(special.struve(0, 25.765368073883174), 4.381964346416636e-07)
```

Comment:
> [WolframAlpha](https://www.wolframalpha.com/input?i=struveH%280%2C+25.765368073883174%29) gives a different result, 4.38196434628207e-07, but I don't trust it. I suspect the internal dps isn't set high enough.

Drive-by comment: You have to be careful when entering floating point values in Alpha.  I suspect Alpha treats `25.765368073883174` as the rational number  `25765368073883174/1000000000000000`, which is not the same value as the double precision value one gets with `25.765368073883174`.  One way to get a better match with Alpha is to convert the floating point value to a rational number via `fractions.Fraction(x)` and then use that rational number in Alpha.  So:

```
In [23]: x = 25.765368073883174

In [24]: Fraction(x)
Out[24]: Fraction(3626153189268873, 140737488355328)
```
Then [Wolfram Alpha](https://www.wolframalpha.com/input?i=struveH%280%2C+3626153189268873%2F140737488355328%29) gives `4.3819643460490980682461752414336891290410805882247547774280... × 10^-7`, which matches your mpmath calculation:

```
In [35]: y = mp.nsum(lambda n: term(0, 25.765368073883174, n), [0, mp.inf])

In [36]: with mp.workdps(58):
    ...:     print(y)
    ...: 
0.0000004381964346049098068246175241433689129041080588224754777428
```

Reply:
Thanks @WarrenWeckesser, that's very helpful. In hindsight, Wolfram's convention of having decimal literals be treated as exact rational numbers is obviously correct since it works with arbitrary precision by default. `mpmath`s is the same. I found that  following matches the result I got from Wolfram Alpha earlier

```python
mp.struveh(0, mp.mpf("25.765368073883174"))
```